data/org.freedesktop.portal.OpenURI.xml

```diff
@@ -163,6 +163,20 @@
       <arg type="o" name="handle" direction="out"/>
     </method>
 
+    <!--
+        SchemeSupported:
+        @scheme: The URI protocol scheme to check
+        @supported: True if the URI scheme is supported
+
+        Checks if OpenURI supports the protocol scheme @scheme.
+
+        The SchemeSupported method was introduced in version 5 of the OpenURI portal API.
+    -->
+    <method name="SchemeSupported">
+      <arg type="s" name="scheme" direction="in"/>
+      <arg type="b" name="supported" direction="out"/>
+    </method>
```

Comment:
You might want to add an options `a{sv}` parameter (with no supported keys for now). More often than not, we notice that we'd like to extend a method at some point.